### ISSUE TITLE
Google Fonts Issue

### ISSUE THREAD

**Issue by Btibert3** [mkearney/mizzourahmd#1]
I just wanted to get a sense of what the theme looked like, but I got the following error:

```
pandoc: Could not fetch https://fonts.googleapis.com/css?family=Roboto:400,400i,700,700i,900
TlsExceptionHostPort (HandshakeFailed (Error_Misc "user error (unexpected type received. expecting handshake and got: Alert [(AlertLevel_Fatal,HandshakeFailure)])")) "fonts.googleapis.com" 443
Error: pandoc document conversion failed with error 67
Execution halted
```

```
---
title: "Title"
author: "Author"
date: "`r Sys.Date()`"
output: mizzourahmd::mizzou_document
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(echo = TRUE, collapse = TRUE, comment = "#>")
```

## R Markdown

This is an R Markdown document. Markdown is a simple formatting syntax for authoring HTML, PDF, and MS Word documents. For more details on using R Markdown see <http://rmarkdown.rstudio.com>.

When you click the **Knit** button a document will be generated that includes both content as well as the output of any embedded R code chunks within the document. You can embed an R code chunk like this:

```{r cars}
summary(cars)
```

## Including Plots

You can also embed plots, for example:

```{r pressure, echo=FALSE}
plot(pressure)
```

Note that the `echo = FALSE` parameter was added to the code chunk to prevent printing of the R code that generated the plot.
```

**Comment by mkearney** [comment]
I believe this is related to your version of Pandoc. That error went away for me around 2.2 (can't remember for sure). You can see your current version with

``` r
> rmarkdown::pandoc_version()
[1] '2.2.3.2'
```

On my mac the solution was to set an environment variable to make sure Rstudio found the right pandoc. So a line in my `~/.Renviron` file looks like;

``` 
RSTUDIO_PANDOC=/usr/local/bin
```

**Comment by mkearney** [comment]
The README file now includes screen shots. ef564dad15e0bdd2db4e1ca7e5fdec0c088af1c7

**Comment by Btibert3** [comment]
Thanks for this.  What was giving me fits is the line above in `Renviron`.  It was telling me that I didn't have pandoc installed, but I did.  I removed that line from the file above, but updated pandoc via `brew` and that did the trick.  What's interesting is that I attempted to update via the terminal within Rstudio, but that didn't work, I had to go to my actual terminal for the update to take place.
